go/ops.go

```diff
@@ -20,38 +20,56 @@ import (
 	_ "golang.org/x/crypto/ripemd160" //nolint:staticcheck
 )
 
-// validate the IAVL Ops
-func validateIavlOps(op opType, b int) error {
+// validateIavlOps validates the prefix to ensure it begins with
+// the height, size, and version of the IAVL tree. Each varint must be a bounded value.
+// In addition, the remaining bytes are validated to ensure they correspond to the correct
+// length.
+func validateIavlOps(op opType, layerNum int) error {
```

Comment:
`b` -> `layerNum` open to other naming suggestions

Reply:
`layerNum` is good to me, maybe we can indicate in godoc that it corresponds to tree depth